packages/tailwindcss-colors/0.1.0/package.yml

```diff
@@ -0,0 +1,747 @@
+matches:
+  - trigger: ":slate50"
+    replace: "#f8fafc"
+matches:
```

Comment:
I'm afraid all instances of `matches:` apart from the first one need to be removed, The package won't work with this duplicated,